src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -64,50 +62,92 @@ public class Intake extends SubsystemBase {
               new MagnetSensorConfigs()
-                  .withMagnetOffset(0.001)
+                  .withMagnetOffset(0.085)
                   .withSensorDirection(SensorDirectionValue.Clockwise_Positive));
 
   public boolean up = false;
   public boolean stop = true;
 
+  private static final CANdiConfiguration intakeSensorsConfig =
+      new CANdiConfiguration()
+          .withDigitalInputs(
+              new DigitalInputsConfigs()
+                  .withS1CloseState(S1CloseStateValue.CloseWhenNotFloating)
+                  .withS1FloatState(S1FloatStateValue.FloatDetect)
+                  .withS2CloseState(S2CloseStateValue.CloseWhenNotFloating)
+                  .withS2FloatState(S2FloatStateValue.FloatDetect));
+
+  private static final TalonFXConfiguration intakeConfig =
+      new TalonFXConfiguration()
+          .withCurrentLimits(
+              new CurrentLimitsConfigs()
+                  .withStatorCurrentLimitEnable(true)
+                  .withStatorCurrentLimit(30))
+          .withSlot0(new Slot0Configs().withKS(5.4).withKP(3));
+
+  private static final NeutralOut motorStop = new NeutralOut();
+  private static final VelocityTorqueCurrentFOC intakeSpin =
+      new VelocityTorqueCurrentFOC(0).withAcceleration(IntakeConstants.intakeAcceleration);
+
+  private static final PositionTorqueCurrentFOC intakeMove = new PositionTorqueCurrentFOC(0);
+
   public Intake() {
     rotation.setNeutralMode(NeutralModeValue.Brake);
     intakeLeft.setNeutralMode(NeutralModeValue.Coast);
     intakeRight.setNeutralMode(NeutralModeValue.Coast);
 
+    intakeLeft.getConfigurator().apply(intakeConfig);
+    intakeRight.getConfigurator().apply(intakeConfig);
+
     rotation.getConfigurator().apply(intakeRotationConfig);
     rotationEncoder.getConfigurator().apply(intakeRotationSensorConfig);
 
     NamedCommands.registerCommand(
-        "IntakeDown", new IntakeMoveCommand(this, false, IntakeConstants.intakeDownAngle));
+        "IntakeDown", new IntakeMoveCommand(this, false, IntakeConstants.intakeDownAngle, 1, -3));
+    intakeSensors.getConfigurator().apply(intakeSensorsConfig);
   }
 
   public double getRotationAngle() {
     return rotation.getRotorPosition().getValueAsDouble() * 360.0;
   }
 
-  public void setAngle(double angle, int slot) {
-    rotation.setControl(new PositionVoltage(angle).withSlot(slot));
+  public double getRotationVelocity() {
+    return rotation.getVelocity().getValueAsDouble();
   }
 
-  public void setIntakeSpeed(double speed) {
-    intakeLeft.setControl(new VoltageOut(speed));
-    intakeRight.setControl(new VoltageOut(-speed));
+  public void setAngle(double angle, int slot, double feedForward) {
+    rotation.setControl(intakeMove.withPosition(angle).withSlot(slot).withFeedForward(feedForward));
+  }
+
+  public void setIntakeSpeed(double velocity) {
+    intakeLeft.setControl(intakeSpin.withVelocity(velocity));
+    intakeRight.setControl(intakeSpin.withVelocity(-velocity));
```

Comment:
I'm wondering if we should make our intake motors spin in opposite directions so all positive movement is intaking and all negative is rejecting. We do have a Ks value for the intake motors and they do use the velocity sign by default, so I'm wondering if we're getting slightly different speeds accidentally too.